app/__init__.py

```diff
@@ -9,7 +9,7 @@
 from flask.ext.wtf import CsrfProtect
 from flask.ext.compress import Compress
 from config import config
-from assets import app_css, app_js, vendor_css, vendor_js
+from assets import app_css, app_js, vendor_css, vendor_js, iThing_css, jqueryui_js, jQDateRangeSlidermin_js
```

Comment:
what might be cleaner would be `vendor.css` and `vendor.js` in their respective folders